src/main/java/de/unistuttgart/t2/creditinstitute/exceptions/PaymentFailedException.java

```diff
@@ -2,10 +2,12 @@
 
 /**
  * Indicates that a payment failed.
- * 
+ *
  * @author maumau
  */
-public class PaymentFailedException extends Exception {
+public final class PaymentFailedException extends Exception {
+
+    private static final long serialVersionUID = 1L;
```

Comment:
I know the formatter did this, and it`s probably not that important, but I'm not sure if 1 is a good serialVersionUID

Reply:
See https://github.com/t2-project/uibackend/pull/6#discussion_r917362361